index.html

```diff
@@ -45,7 +45,7 @@
     <p>In addition to matching you to the eight values, the quiz also attempts to match you to a political ideology. This is a work in progress and is much less accurate than the values and axes, so don't take it too seriously. If you disagree with your assigned ideology, send us an email at eightvalues@gmail.com with your scores, matched ideology, and preferred ideology, and we'll look into adjusting the system. Thanks!</p>
 <h2>I don't like my scores!</h2>
     <p>¯\_(ツ)_/¯<br/>
-    If you have any suggestions or constructive criticism, feel free to send it to eightvalues@gmail.com or open an issue on the GitHub page!</p>
+    If you have any suggestions or constructive criticism, feel free to send it to eightvalues@gmail.com or open an issue on the GitHub page here: <a href="https://github.com/8values/8values.github.io">8values.GitHub</a></p>
```

Comment:
Could you also add a link for the eMail?
`<a href="mailto:eightvalues@gmail.com">eightvalues@gmail.com</a>`
also I would simply do it like this:
_… issue on the_ `<a href="https://github.com/8values/8values.github.io">GitHub page</a> `